types/trusted-types/test/browser.ts

```diff
@@ -106,3 +100,10 @@ trustedHTML = trustedScript;
 
 // $ExpectError
 new TrustedHTML();
+
+// $ExpectType typeof TrustedHTML
```

Comment:
please also add the ones for `window.Trusted*`

Reply:
Done.